index.js

```diff
@@ -193,6 +201,24 @@ module.exports = function (content) {
     this.addDependency(generatorOptions.cssFontsUrl);
   }
 
+  const isCSSLoader = l => /(\/|\\|@)css-loader/.test(l.path);
+  const cssLoaderIndex = this.loaders.findIndex(isCSSLoader);
+
+  if (cssLoaderIndex > -1) {
+    const cssLoaderOptions = this.loaders[cssLoaderIndex].options || {};
+    const cssLoaderUrlOption = cssLoaderOptions.url;
+
+    cssLoaderOptions.url = (url, resourcePath) => {
+      if (resourcePath === this.resourcePath) {
+        return false;
+      }
+
+      return cssLoaderUrlOption;
+    };
+
+    this.loaders[cssLoaderIndex].options = cssLoaderOptions;
+  }
```

Comment:
Bad idea

Reply:
I guess it should be done in the pitch phase, but other than that, what's so bad about it?

Reply:
Reaching into the configuration object of another module and mutating it is generally unexpected and fragile. The user might be using the `url` option for something else, or the object might be aliased elsewhere in the configuration. Or your heuristic might work half of the time and leave the user wondering why one thing works and another thing fails. Better to document what the user needs to configure.

Reply:
Also, if nothing else, this configuration change has little to do with Webpack 5 support; the same issue is present with Webpack 4. So a separate PR would be better.

Reply:
I noticed I can just use `url: () => false`, as the `css-loader` is in a rule specific to `font.js` files.